react-carousel-component/my-app/src/Carousel.jsx

```diff
@@ -0,0 +1,48 @@
+import { useEffect } from 'react';
+import { useState } from 'react';
+
+export default function Carousel({ images }) {
+  const [pokemonIndx, setPokemonIndx] = useState(0);
```

Comment:
To make this more reusable, you probably shouldn't have the word "pokemon" everywhere. It's fun, though!